src/components/SelectionList/RadioListItem.tsx

```diff
@@ -41,6 +41,7 @@ function RadioListItem<TItem extends ListItem>({
             pendingAction={item.pendingAction}
         >
             <>
+                {!!item.leftElement && item.leftElement}
```

Comment:
If you would pass an empty string it would render an empty string. And in the code it looks consistent - look at the item.rightElement or item.alternateText